code/modules/mob/living/carbon/human/species/working_joe/_species.dm

```diff
@@ -39,8 +35,6 @@
 
 // Special death noise for Working Joe
 /datum/species/synthetic/colonial/working_joe/handle_death(mob/living/carbon/human/dying_joe, gibbed)
-	if(!gibbed) //A gibbed Joe won't have a death rattle
```

Comment:
shouldn't these just have been `if(gibbed)` instead?

Reply:
No, i've investigated it and gibbed argument never procs for sytnths